merino/curated_recommendations/provider.py

```diff
@@ -197,6 +198,16 @@ def is_fakespot_experiment(request, surface_id) -> bool:
             and surface_id == ScheduledSurfaceId.NEW_TAB_EN_US
         )
 
+    @staticmethod
+    def is_follow_unfollow_block_section_experiment(request, surface_id) -> bool:
+        """Check if the Follow/Unfollow & Block Sections experiment is enabled."""
+        return (
+            request.feeds
+            and "sections" in request.feeds
+            and request.sections
+            and surface_id == ScheduledSurfaceId.NEW_TAB_EN_US
+        )
```

Comment:
```suggestion
        return bool(request.sections)
```

I would recommend letting Nimbus control which regions are enrolled in the experiment, and only check that the request contains the required parameters for us to serve it successfully. I think we can simply check if `request.sections` is truthy and whether there are sections.